tests/test_encode_decode.py

```diff
@@ -95,8 +96,80 @@ def test_b85_encode_decode_round_trip(self, payload, pad):
 
 
 class TestBinASCII(unittest.TestCase):
-    # TODO: https://docs.python.org/3/library/binascii.html
-    pass
+    @given(payload=st.binary(), backtick=st.booleans())
+    def test_b2a_uu_a2b_uu_round_trip(self, payload, backtick):
+        x = binascii.b2a_uu(payload, backtick=backtick)
+        self.assertEqual(payload, binascii.a2b_uu(x))
+
+    @given(payload=st.binary(), newline=st.booleans())
+    def test_b2a_base64_a2b_base64_round_trip(self, payload, newline):
+        x = binascii.b2a_base64(payload, newline=newline)
+        self.assertEqual(payload, binascii.a2b_base64(x))
+
+    @given(
+        payload=st.binary(),
+        quotetabs=st.booleans(),
+        istext=st.booleans(),
+        header=st.booleans(),
+    )
+    def test_b2a_qp_a2b_qp_round_trip(self, payload, quotetabs, istext, header):
+        x = binascii.b2a_qp(payload, quotetabs=quotetabs, istext=istext, header=header)
+        self.assertEqual(payload, binascii.a2b_qp(x, header=header))
+
+    @given(payload=st.binary())
+    def test_rlecode_hqx_rledecode_hqx_round_trip(self, payload):
+        x = binascii.rlecode_hqx(payload)
+        self.assertEqual(payload, binascii.rledecode_hqx(x))
+
+    @given(payload=st.binary())
+    def test_b2a_hqx_a2b_hqx_round_trip(self, payload):
+        # assuming len(payload) as 3, since it throws exception: binascii.Incomplete, when length is not a multiple of 3
+        assume(len(payload) % 3 == 0)
+        x = binascii.b2a_hqx(payload)
+        res, _ = binascii.a2b_hqx(x)
+        self.assertEqual(payload, res)
+
+    @given(payload=st.binary(), value=st.just(0) | st.integers())
+    def test_crc_hqx(self, payload, value):
+        crc = binascii.crc_hqx(payload, value)
+        self.assertIs(type(crc), int)
+
+    @given(
+        payload_piece_1=st.binary(),
+        payload_piece_2=st.binary(),
+        value=st.just(0) | st.integers(),
+    )
+    def test_crc_hqx_two_pieces(self, payload_piece_1, payload_piece_2, value):
+        combined_crc = binascii.crc_hqx(payload_piece_1 + payload_piece_2, value)
+        crc = binascii.crc_hqx(payload_piece_1, value)
+        crc = binascii.crc_hqx(payload_piece_2, crc)
```

Comment:
```suggestion
        crc_part1 = binascii.crc_hqx(payload_piece_1, value)
        crc = binascii.crc_hqx(payload_piece_2, crc_part1)
```

make it clear that we're not discarding the variable we just defined.